index.js

```diff
@@ -1,14 +1,10 @@
 import Transport from "winston-transport";
-import { KafkaClient, HighLevelProducer } from "kafka-node";
 import { LEVEL, MESSAGE } from "triple-beam";
-
-const noop = () => {};
+import promisify from "promisify-node";
```

Comment:
Do we want to support older nodes? Since 8 we already have promisify :)

Reply:
But this could be used in the browser, right?

Reply:
promisify-node is a node lib. It relies on require. It also does some "complex" stuff to detect callback params. It's overkill IMHO.

Reply:
oops, ok!